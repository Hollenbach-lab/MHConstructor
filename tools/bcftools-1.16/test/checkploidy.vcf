##fileformat=VCFv4.1
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=22,assembly=b37>
##contig=<ID=X,assembly=b37>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	S1
22	1	1	C	G	.	PASS	.	GT	1/1
X	1	2	C	G	.	PASS	.	GT	1
