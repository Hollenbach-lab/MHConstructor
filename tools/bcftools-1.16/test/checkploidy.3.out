# [1]Sample	[2]Chromosome	[3]Region Start	[4]Region End	[5]Ploidy
S1	20	1	1	2
S1	21	1	1	2
S1	22	1	1	2
S1	X	1	1	1
